modules/user-guide/pages/installing-vale-cli.adoc

```diff
@@ -19,6 +19,20 @@ The project will already have a `vale.ini` configuration.
 
 * Install the link:https://docs.asciidoctor.org/asciidoctor/latest/install/[Asciidoctor] Ruby gem.
 
+* If you have previously installed Vale, you can check what version you are currently running.
+Run the following command:
++
+[source,terminal]
+----
+$ vale --version
+----
++
+.Example output
+[source,terminal]
+----
+vale version 3.6.1
+----
+
```

Comment:
I think this is good information. However, it might not match the experience of Red Hat users who follow the https://vale.sh/docs/vale-cli/installation/ instructions for Linux users, which have us install Vale by using `snap`. 
In that case users see:
```
[rdlugyhe@rdlugyhe-thinkpadt14sgen2i ~]$ which vale
/var/lib/snapd/snap/bin/vale
[rdlugyhe@rdlugyhe-thinkpadt14sgen2i ~]$ vale --version
vale version master
```

Reply:
The snap package manager automatically gets and runs the latest version of the package being run by the user.

Reply:
Snap is an additional package manager does is not installed by default in Fedora OS. Seems strange that the vale install does not report the version though. That is odd.

Also I just installed snap on F40, and got this error: 

```
$ sudo snap install vale
error: too early for operation, device not yet seeded or device model not acknowledged
```

For RHEL and Fedora, see https://redhat-documentation.github.io/vale-at-red-hat/docs/main/user-guide/installing-vale-cli/